public_html/wp-content/plugins/wcpt/wcpt-event/class-event-admin.php

```diff
@@ -402,15 +404,19 @@ public function display_post_states( $states ) {
 	/**
 	 * Save metadata from form
 	 *
-	 * @param int     $post_id Post ID.
-	 * @param WP_Post $post    Post Object.
+	 * @hook save_post
 	 */
 	public function metabox_save( $post_id, $post ) {
 		// Don't add/remove meta on revisions and auto-saves.
 		if ( wp_is_post_autosave( $post_id ) || wp_is_post_revision( $post_id ) ) {
 			return;
 		}
 
+		// Make sure the requset came from the edit post screen.
+		if ( empty( $_POST['_wpnonce'] ) || ! wp_verify_nonce( $_POST['_wpnonce'], 'update-post_' . $post_id ) ) {
```

Comment:
- [ ] Test if `metabox_save` is working, and not working if nonce is not verified.